apps/src/templates/teacherDashboard/CollapsibleSection.story.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import CollapsibleSection from './CollapsibleSection';
+import CourseCard from './CourseCard';
+import SetUpMessage from './SetUpMessage';
+
+const EXAMPLE_CARD_DATA = {
+  courseName: "CSP Unit 2 - Digital Information",
+  description: "Explore how more complex digital information is represented and manipulated through computation and visualization",
+  image: "this is where there will be the source for the photo",
+  link: "link to wherever you want the button to go...",
```

Comment:
The links and images right now are just placeholder notes to myself. Once the components are hooked up to a page and I've done the dynamic require for images (when we know what those images will be), I'll come back and change these.